www/js/components/add-point-card.js

```diff
@@ -54,12 +54,12 @@ export class AddPointCard extends Component {
   }
 
   onSubmit() {
-    const { mapState } = this.props;
+    const { center } = this.props;
```

Comment:
I think it is more meaningful if we do <AddPointCard location={[lat, lng]} /> This component shouldn't care that <PointMap /> gets that location from the center of its map.